sdk/ai/azure-ai-generative/azure/ai/generative/_telemetry/logging_handler.py

```diff
@@ -21,13 +21,12 @@
 )
 from opencensus.trace import config_integration
 
-from azureml.telemetry import INSTRUMENTATION_KEY
-
 from azure.ai.ml._telemetry.logging_handler import in_jupyter_notebook, CustomDimensionsFilter
 
 from azure.ai.generative._user_agent import USER_AGENT
 
 GEN_AI_INTERNAL_LOGGER_NAMESPACE = "azure.ai.generative._telemetry"
+INSTRUMENTATION_KEY = "71b954a8-6b7d-43f5-986c-3d3a6605d803"
```

Comment:
This can be a follow up. But given ai-generative package depends on ai-resources package, can we consolidate the telemetry logic in one package and reduce duplicate code?

Reply:
Most logging things that can be easily shared are already placed in azure.ai.ml and imported to ai.resources and ai.generative. I agree that INSTRUMENTATION_KEY falls into that category and can be moved, so I can do that here. Previously, it was being imported to each via azureml.telemetry but Ankit asked on this PR that I remove that dependency.

There are some methods that could be abstracted a bit to be modularized and then each SDK passes in args. I can make that a backlog item and review it with you.

Reply:
Just moved INSTRUMENTATION_KEY to ai.ml only